processor/configuration/src/main/java/com/redhat/service/smartevents/processor/GatewayConfiguratorImpl.java

```diff
@@ -0,0 +1,100 @@
+package com.redhat.service.smartevents.processor;
+
+import java.util.Collection;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+
+import com.redhat.service.smartevents.infra.exceptions.definitions.user.GatewayProviderException;
+import com.redhat.service.smartevents.infra.models.gateways.Action;
+import com.redhat.service.smartevents.infra.models.gateways.Source;
+
+@ApplicationScoped
+public class GatewayConfiguratorImpl implements GatewayConfigurator {
```

Comment:
similar comment as above

Reply:
Yes we can think about this after Event Sources are implemented :+1:

Reply:
Created https://issues.redhat.com/browse/MGDOBR-632